modules/tribes/tests/client/components/TribesPage.component.tests.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import { within } from '@testing-library/react';
+import { render, fireEvent, waitForElement } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import '@/config/client/i18n';
+
+import TribesPage from '@/modules/tribes/client/components/TribesPage.component';
+import * as tribesApi from '@/modules/tribes/client/api/tribes.api';
```

Comment:
FYI we called this `tribesApi` instead of `tribes` because the `eslint-plugin-import` was complaining when we used the `tribes` name later on.